server/routes/api/place.py

```diff
@@ -212,7 +212,7 @@ def statsvars(dcid):
 
 
 @cache.memoize(timeout=3600 * 24)  # Cache for one day.
-def get_stat_vars_union(dcids):
+def get_stat_vars_union(dcids, stat_vars):
```

Comment:
unfortunately we use "dcids" in the REST api; Here for clarity, should change "dcids" to "places"

Reply:
done